src/index.ts

```diff
@@ -1,21 +1,35 @@
-import { connectToDatabase } from './config/dbconnection';
 import express from "express";
 import 'dotenv/config'
+import cors from "cors";
+import bodyParser from 'body-parser';
+import { connectToDatabase, initializeCategories } from './config/dbconnection';
+import userRouter from './routers/userRouter';
+import categoryRouter from './routers/CategoryRouter';
 
 const port = process.env.SWE_BACKEND_PORT || 5000;
+const corsOptions = {
+    origin: `http://localhost:${process.env.SWE_FRONTEND_PORT}`,
```

Comment:
That basically denies responses unless, they come from this origin, right?

Reply:
yes, by default cross-origin requests are rejected and this is a whitelist

Reply:
Ok all clear!